### PR TITLE
Fix CCIP Load Tests in Core Repo

### PR DESCRIPTION
**Pull request by kalverra**

### CODE REVIEW

File: .github/workflows/ccip-load-tests.yml [hunk 1]
```diff
@@ -189,13 +190,16 @@ jobs:
           RR_CPU: 4
           TEST_TRIGGERED_BY: ccip-load-test-ci-${{ matrix.type.name }}
           BASE64_CCIP_CONFIG_OVERRIDE: ${{ steps.set_override_config.outputs.base_64_override }},${{ steps.setup_create_base64_config_ccip.outputs.base64_config }}
-          TEST_BASE64_CCIP_CONFIG_OVERRIDE: ${{ steps.set_override_config.outputs.base_64_override }},${{ steps.setup_create_base64_config_ccip.outputs.base64_config }}          
+          TEST_BASE64_CCIP_CONFIG_OVERRIDE: ${{ steps.set_override_config.outputs.base_64_override }},${{ steps.setup_create_base64_config_ccip.outputs.base64_config }}
```
Comment (b-gopalswami) [inline .github/workflows/ccip-load-tests.yml]: Not in this PR but when you pass the override config, this is not getting applied properly unless we swap this.
```suggestion
          TEST_BASE64_CCIP_CONFIG_OVERRIDE: ${{ steps.setup_create_base64_config_ccip.outputs.base64_config }},${{ steps.set_override_config.outputs.base_64_override }}
```

File: integration-tests/scripts/entrypoint [hunk 2]
```diff
@@ -9,6 +9,8 @@ SCRIPT_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)
 
 cd "$SCRIPT_DIR"/../ || exit 1
 
+ls
```
Comment (b-gopalswami) [inline integration-tests/scripts/entrypoint]: Do we need this?

File: integration-tests/test.Dockerfile [hunk 3]
```diff
@@ -2,7 +2,7 @@ ARG BASE_IMAGE
 ARG IMAGE_VERSION=latest
 FROM ${BASE_IMAGE}:${IMAGE_VERSION} AS build-env
 
-ARG SUITES=chaos migration performance reorg smoke soak benchmark
+ARG SUITES=chaos migration performance reorg smoke soak benchmark load
```
Comment (b-gopalswami) [inline integration-tests/test.Dockerfile]: Do we need to add "load" to this list?

**Review by b-gopalswami — COMMENTED** [review]
Thanks @kalverra for fixing this! Just few questions ow it LGTM!

File: integration-tests/scripts/entrypoint (hunk 2 above)
Comment (kalverra) [reply integration-tests/scripts/entrypoint]: Nope, debug, will remove

File: integration-tests/test.Dockerfile (hunk 3 above)
Comment (kalverra) [reply integration-tests/test.Dockerfile]: Probably not, can remove

**Review by b-gopalswami — DISMISSED** [review]

File: .github/workflows/ccip-load-tests.yml [hunk 4]
```diff
@@ -127,10 +128,11 @@ jobs:
           - name: stable-load
             run: ^TestLoadCCIPStableRPS$
             os: ubuntu-latest
-          - name: load-with-arm-curse-uncurse
-            run: ^TestLoadCCIPStableRPSAfterARMCurseAndUncurse$
-            config_path: ./integration-tests/ccip-tests/testconfig/tomls/load-with-arm-curse-uncurse.toml
-            os: ubuntu-latest
+          # Enable when CCIP-2277 is resolved
```
Comment (AnieeG) [inline .github/workflows/ccip-load-tests.yml:131]: is this still an issue?

File: integration-tests/test.Dockerfile [hunk 5]
```diff
@@ -13,6 +13,7 @@ FROM ${BASE_IMAGE}:${IMAGE_VERSION}
 RUN mkdir -p /go/testdir/integration-tests/scripts
 COPY --from=build-env /go/pkg /go/pkg
 COPY --from=build-env /go/testdir/integration-tests/*.test /go/testdir/integration-tests/
+COPY --from=build-env /go/testdir/integration-tests/ccip-tests/*.test /go/testdir/integration-tests/
```
Comment (AnieeG) [inline integration-tests/test.Dockerfile:16]: needed?

File: integration-tests/testconfig/default.toml [hunk 6]
```diff
@@ -322,11 +322,11 @@ eip_1559_dynamic_fees = true
...
 # automated gas estimation for live networks
 # if set to true we will dynamically estimate gas for every transaction (based on suggested values, priority and congestion rate for last X blocks)
- gas_price_estimation_enabled = true
+gas_price_estimation_enabled = true
 # number of blocks to use for congestion rate estimation (it will determine buffer added on top of suggested values)
- gas_price_estimation_blocks = 100
+gas_price_estimation_blocks = 100
 # transaction priority, which determines adjustment factor multiplier applied to suggested values (fast - 1.2x, standard - 1x, slow - 0.8x)
- gas_price_estimation_tx_priority = "standard"
+gas_price_estimation_tx_priority = "standard"
```
Comment (AnieeG) [inline integration-tests/testconfig/default.toml:329]: related?

Comment (kalverra) [reply integration-tests/testconfig/default.toml:329]: Unrelated linting fix

File: integration-tests/test.Dockerfile (hunk 5 above)
Comment (kalverra) [reply integration-tests/test.Dockerfile:16]: Yes, this actually pulls the tests that are built in CCIP into the runner image. Without it we just have core load tests (or none).

File: .github/workflows/ccip-load-tests.yml (hunk 4 above)
Comment (kalverra) [reply .github/workflows/ccip-load-tests.yml:131]: According to the CCIP JIRA board, yes

**Review by AnieeG — APPROVED** [review]

**Review by lukaszcl — APPROVED** [review]

**Pull request merged**